openfisca_us/variables/demographic/person.py

```diff
@@ -219,3 +219,31 @@ def formula(person, period):
                 DurationOfCare.HOURLY,
             ],
         )
+
+
+class permanently_disabled_veteran(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = (
+        "Indicates whether a person is a permanently disabled veteran"
+    )
+    label = "Permanently Disabled Veteran"
```

Comment:
only capitalize first word, here and elsewhere (unless a proper noun like Social Security Disability Insurance)